contracts/SCERC721Derivative.sol

```diff
@@ -59,4 +59,8 @@ contract SCERC721Derivative is ERC721, Ownable {
   {
     return super.supportsInterface(_interfaceId);
   }
+
+  function setVerifierAddress(address _verifier) external onlyOwner {
+    verifier = _verifier;
```

Comment:
should we make `verifier` public?

Reply:
The `verifier` variable is `private` for this file but the `setVerifierAddress` function is `external` and `onlyOwner`, should this be an internal function then?